src/modules/users/users.controller.ts

```diff
@@ -1,11 +1,25 @@
-import { Controller, Get, Param, ParseBoolPipe, ParseIntPipe, Query } from '@nestjs/common'
+import {
+    Controller,
+    Get,
+    Param,
+    ParseBoolPipe,
+    ParseIntPipe,
+    Query,
+    UseGuards,
+} from '@nestjs/common'
+import { AuthGuard } from '@nestjs/passport'
+import { ApiBearerAuth } from '@nestjs/swagger'
 import { Pageable } from 'src/models'
+import { UsersModel } from 'src/models/users'
 import { UsersDto } from 'src/modules/users/dto/users.dto'
 import { UsersFactory } from 'src/modules/users/users.factory'
+import { GetUser } from 'src/shared/decorators/get-user.decorator'
 import { QueryUserDto } from './dto/get-users.dto'
 import { UsersService } from './users.service'
 
 @Controller('users')
+@UseGuards(AuthGuard('jwt'))
```

Comment:
AuthGuard('jwt')
move to guard folder 
@Injectable()
export class JwtAuthGuard extends AuthGuard('jwt') {}